shoes-core/lib/shoes/common/state.rb

```diff
@@ -1,16 +1,30 @@
 class Shoes
   module Common
     module State
```

Comment:
curiously we had this, but it wasn't included anywhere o_O might have gotten lost in the big style PR or somewhere - another reason against big PRs :D